python/mlcroissant/mlcroissant/_src/structure_graph/nodes/metadata.py

```diff
@@ -237,6 +244,19 @@ def validate_license(self) -> list[str] | None:
             self.add_error(f"License should be a list of str. Got: {license}")
             return None
 
+    def validate_date(self, date: Any) -> datetime.datetime | None:
```

Comment:
If time allows maybe add a parametrizxed test?

Reply:
Done.

I did it in `test_from_jsonld` because `from_str_to_datetime` is already parametrized with various inputs. Thanks!